src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -20,4 +24,31 @@ public LineResponse saveLine(LineRequest request) {
         Line persistLine = lineRepository.save(request.toLine());
         return LineResponse.of(persistLine);
     }
+
+    @Transactional(readOnly = true)
+    public List<LineResponse> findAllLines() {
+        List<Line> lines = lineRepository.findAll();
+
+        return lines.stream()
+                .map(line -> LineResponse.of(line))
```

Comment:
인자를 그대로 전달만 한다면 메서드 레퍼런스를 활용하면 간결하게 코드를 작성할 수 있습니다.

```suggestion
                .map(LineResponse::of)
```